DoSomething/bertly
# Provide 404 HTML response for unknown tokens

#### What's this PR do?

This PR updates the response to an unknown shortened token:

- Respond with `404` instead of `400`
- Redirect to [generic DoSomething 404 page](https://next.dosomething.org/us/404)

#### How should this be reviewed?

- Run locally
- Use a browser to request a bad page, such as http://localhost:5000/url-doesnt-exist
- Verify `404` response

#### Any background context you want to provide?

N/A

#### Relevant tickets

#18 

#### Checklist

#### Screenshots & links

404 page: https://next.dosomething.org/us/404
